pkgs/tools/cd-dvd/cue2pops/default.nix

```diff
@@ -5,29 +5,30 @@
 
 stdenv.mkDerivation {
   pname = "cue2pops";
-  version = "unstable-2018-01-04";
+  version = "0-unstable-2023-01-15";
```

Comment:
Is this `0-` thing a new convention? I haven’t seen it before (but I was absent for quite a while).

Reply:
It's a new convention yes. I was recently informed that unstable packages must begin with `0`, but can't find the exact line in contributing guidelines atm.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming